presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -352,7 +352,15 @@ public PlanNode visitSample(SampleNode node, RewriteContext<Void> context)
         @Override
         public PlanNode visitValues(ValuesNode node, RewriteContext<Void> context)
         {
-            return context.defaultRewrite(node);
+            List<List<Expression>> canonicalizedRows = node.getRows().stream()
+                    .map(this::canonicalize)
+                    .collect(toImmutableList());
+            List<Symbol> canonicalizedOutputSymbols = canonicalizeAndDistinct(node.getOutputSymbols());
```

Comment:
cardinality of output symbols must match cardinality of each row, see:
```
com.facebook.presto.sql.planner.plan.ValuesNode#ValuesNode
```
so `canonicalize` should be used